configs/mmcls/classification_sdk_dynamic.py

```diff
@@ -0,0 +1,8 @@
+_base_ = ['./classification_dynamic.py', '../_base_/backends/sdk.py']
+
+codebase_config = dict(model_type='sdk')
+
+pipeline = [
```

Comment:
Better not put it in the root. What about backend_conig? @AllentDan @SingleZombie

Reply:
agree

Reply:
agree

Reply:
fixed